src/do-you-want-to-stop.py

```diff
@@ -1,3 +1,10 @@
 
 # You should write a loop around this question, and keep asking until
 # the answer is "yes" (lower case).
+
+answer = input('Do you want to stop? ')
+
+while answer:
```

Comment:
If `x` is a string, then `while x` will run as long as `x` is not the empty string. This while loop will continue until the user hits enter without anything else on the prompt. It doesn't stop when the user types `"yes"` (that just means that the user doesn't get a prompt because you only display that when `answer != "yes"`).

You want the loop to stop when `answer` is `"yes"`, so you can write `while answer != "yes"`.